exchange/websocket/manager.go

```diff
@@ -397,7 +403,7 @@ func (m *Manager) getConnectionFromSetup(c *ConnectionSetup) *connection {
 		readMessageErrors:        m.ReadMessageErrors,
 		shutdown:                 m.ShutdownC,
 		Wg:                       &m.Wg,
-		Match:                    m.Match,
+		Match:                    match,
```

Comment:
I think this can now be unexported thanks to `(c *connection) RequireMatchWithData`
See diff:
https://gist.github.com/gloriousCode/663741b41fe73fa48c7c60f64bea2499

Reply:
Good idea! There's a couple exchange which have auth and unauth data funneling into the same ws data handler and no real way of differentiating between the two connections from that when it's not using multiconnection ws. I think I might defer this once we do a full upgrade to that across exchanges.